daphne/src/roles_test.rs

```diff
@@ -28,27 +29,67 @@ const TASK_LIST: &str = r#"{
         "min_batch_duration": 3600,
         "min_batch_size": 100,
         "vdaf": {
-            "prio3": {
-                "histogram": {
-                    "buckets": [77, 999]
-                }
-            }
+            "prio3": "count"
         },
         "vdaf_verify_key": "1fd8d30dc0e0b7ac81f0050fcab0782d"
     }
 }"#;
 
+const HPKE_CONFIG_LIST: &str = r#"[
+    "1700200001000100205dc71373c6aa7b0af67944a370ab96d8b8216832579c19159ca35d10f25a2765",
+    "0e0020000100010020b07126295bcfcdeaec61b310fd7ffbf8c6ca7f6c17e3e0a80a5405a242e5084b"
+]"#;
+
+const HPKE_SECRET_KEY_LIST: &str = r#"[
+    {
+        "id": 23,
+        "sk": "888e94344585f44530d03e250268be6c6a5caca5314513dcec488cc431486c69"
+    },
+    {
+        "id": 14,
+        "sk": "b809a4df399548f56c3a15ebaa4925dd292637f0b7e2f6bc3ba60376b69aa05e"
+    }
+]"#;
```

Comment:
I was wondering whether there was any reason why `Copy` or `Clone` was not added (e.g., to prevent any copying / cloning of secret keys?).
In any case, I can implement that and the `HpkeReceiverConfig` struct.

Reply:
As a general rule it's best to avoid implementing functionality that's not exercised. Typically if you need `Clone`, you should feel free to add it :)

Reply:
I've implemented the `HpkeReceiverConfig` struct and used it to replace the `HPKE_CONFIG_LIST` and `HPKE_SECRET_KEY_LIST` in `roles_test.rs`.
There are several changes needed to support this, mainly adding `#[serde(with = "hex")]` to the `HpkeConfig` struct so that serde is able to parse public keys encoded in hex.